pkgs/development/libraries/freetype/default.nix

```diff
@@ -49,7 +49,7 @@ in stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
-  configureFlags = [ "--disable-static" "--bindir=$(dev)/bin" ];
+  configureFlags = [ "--disable-static" "--bindir=$(dev)/bin" "--enable-freetype-config" ];
```

Comment:
It looks like freetype's build system is also installing a pkg-config file, and I assume that if `freetype-config` is going away, downstream packages should be checking for freetype via pkg-config? How many packages in nixpkgs depend on `freetype-config`? Should you be creating additional issues to track or fix those, or leaving a comment here so we know when to remove this flag?

Reply:
@endgame everything you said seems correct. i'm not sure now to track what packages currently use `freetype-config` other than testings all dependencies without including the binary and seeing what breaks

Reply:
Maybe make a branch that's the version bump with no configure argument, and get someone to point hydra at it? I don't know who the correct person is, so I'm going to take a stab and ping @grahamc since ofborg has his name on things.

@grahamc hopefully you can add the right people and then unsub yourself. If my guess was wildly out-of-line, then I'm sorry.

Reply:
@endgame see #52524